modules/ROOT/pages/siebel/siebel-connector.adoc

```diff
@@ -154,15 +154,48 @@ Displays all records that includes a valid owner. For example, it includes the r
 [NOTE]
 Limit the number of objects to retrieve through DataSense to a few objects using search specifications, otherwise retrieving metadata slows down Studio.
```

Comment:
```suggestion
Use search specifications to limit the number of objects to retrieve through DataSense to a few objects; otherwise, retrieving metadata slows down Studio.
```